src/NodeChanger/MethodNameChanger.php

```diff
@@ -10,12 +10,29 @@
 {
     /**
      * @param string[] $renameMethodMap
+     * @param string[]|string[] $acceptedAssertionMethods
      */
-    public function renameNode(MethodCall $node, array $renameMethodMap): ?Node
-    {
+    public function renameNode(
+        MethodCall $node,
+        array $renameMethodMap,
+        array $acceptedAssertionMethods = [],
```

Comment:
What is this property exactly? It should not be related to PHPUnit at all, I suppose it is reference here :)

Reply:
If you are not able to make it PHPUnit indepenent, creating `PHPUnitAssertMethodNameChager` class would be better to decouple.
It clearly state the intent of the class